howso/utilities/installation_verification.py

```diff
@@ -422,26 +446,23 @@ def generate_dataframe(*, client: AbstractHowsoClient,
         }
     }
     feature_names = list(features.keys())
-    context_features = list(feature_names)[:-1]
```

Comment:
I wonder if these context_features should have been passed into react given recent changes to removal of default_context_features.